docs/getting-started.md

```
@@ -75,34 +86,62 @@ issue.
+```erb
+<%= react_component("HelloWorld", @hello_world_props, auto_load_bundle: true) %>
+<%= react_component("HeavyMarkdownEditor", @editor_props, auto_load_bundle: true) %>
+```
+
+With `auto_load_bundle: true`, and by placing your "exposed" components in the appropriate directories, React on Rails:
+
+- Automatically finds and bundles your component.
+- Automatically registers it for use.
+- Eliminates the need for manual pack configuration.
+
+See [Auto-Bundling: File-System-Based Automated Bundle Generation][./guides/auto-bundling-file-system-based-automated-bundle-generation.md]
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "./guides/auto-bundling-file-system-based-automated-bundle-generation.md"

(MD052, reference-links-images)